2-promisses/http-2-promise.ts

```diff
@@ -32,7 +55,7 @@ function httpRequst(): Promise<string> {
 httpRequst().then(function(value) {
     console.log(value)
 })
-
+*/
 console.log("Data received");
 
 // Zadanie: Wykorzystaj składnie async await i zmodyfikuj powyższy kod
```

Comment:
I think you were only suppose to change this part:

httpRequst().then(function(value) {
    console.log(value)
})